src/tiled/commanddialog.cpp

```diff
@@ -76,17 +83,56 @@ void CommandDialog::setShortcut(const QKeySequence &keySequence)
         mUi->treeView->model()->setShortcut(current, keySequence);
 }
 
-void CommandDialog::updateKeySequenceEdit(const QModelIndex &current, const QModelIndex &)
+void CommandDialog::setSaveBeforeExecute(int state)
+{
+    const QModelIndex &current = mUi->treeView->currentIndex();
+    if (current.row() < mUi->treeView->model()->rowCount(QModelIndex()))
```

Comment:
You added the default parameter but didn't remove the now no longer necessary argument here. :-)